isis/src/base/objs/RollingShutterCameraDetectorMap/RollingShutterCameraDetectorMap.cpp

```diff
@@ -127,16 +134,11 @@ namespace Isis {
     int iterations = 0;
     int maxIterations = 50;
 
-    p_detectorSample = sample;
-    p_detectorLine   = line;
-    p_parentSample   = (p_detectorSample - p_ss) / p_detectorSampleSumming + 1.0;
-    p_parentLine     = (p_detectorLine   - p_sl) / p_detectorLineSumming   + 1.0;
```

Comment:
Removed these so that the apply and remove jitter functions are idempotent. These are already being set by RollingShutterCameraDetectorMap::SetDetector before calling RollingShutterCameraDetectorMap::applyJitter